app/test_main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from unittest import mock
+
+from app.main import (
+    valid_google_url,
+    can_access_google_page
+)
+
+
+@mock.patch("requests.get")
+def test_valid_google_url(mocked_request: any) -> None:
+    valid_google_url("")
+
+    mocked_request.assert_called_once()
+
```

Comment:
According to the task description you have to check only can_access_google_page functionality.